src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts

```diff
@@ -80,14 +80,19 @@ describe('AvailableTasksComponent', () => {
     expect(component.tasks.length).toEqual(2);
   });
 
-  it('should have all columns, including "Manage +"', () => {
+  it('should have all column headers apart from DerivedIcon, including "Manage +"', () => {
```

Comment:
With the change to the condition below, you don't need to specify "apart from DerivedIcon".